gitbook-docs/market/cmd-reference/user.md

```diff
@@ -1,22 +1,18 @@
 ---
-description: Scripts to manage user
+description: Scripts to manage user authentication
 ---
 
-# user
+# User
 
-User scripts are used to manage user credentials/accounts
+User scripts are used to manage user credentials/accounts.
 
-## New User
+## New user
```

Comment:
```suggestion
# Creating a new account```

Reply:
Fixed

Reply:
Is “User account utilities” fine? As this page will later describe not only signing up, but also signing in and registering as a proof producer

Reply:
Yep, it's a good title

Reply:
or just “Account utilities”